packages/web-components/fast-components/src/tree-item/tree-item.styles.ts

```diff
@@ -94,6 +94,15 @@ export const treeItemStyles: FoundationElementTemplate<ElementStyles, TreeItemOp
     definition
 ) =>
     css`
+    @keyframes treeItemLoading {
```

Comment:
This might be worth adding a code comment to for future readers.

Reply:
I thought about that - I'll add a note here.